packages/core/components/SidebarSection/styles.module.css

```diff
@@ -12,7 +13,7 @@
   background: white;
   padding: 16px;
   border-bottom: 1px solid var(--puck-color-grey-8);
-  overflow-x: scroll;
+  overflow-x: auto;
```

Comment:
Can you split this fix into a separate commit for our changelogs?

```
fix: only show sidebar scroll bars if necessary
```

Reply:
yeah, can do.